ci-operator/step-registry/network/conf/aws-cluster-mtu/network-conf-aws-cluster-mtu-commands.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+CONFIG="${SHARED_DIR}/install-config.yaml"
+PATCH="${ARTIFACT_DIR}/install-config.patch.yaml"
+
+export PATH=${PATH}:/tmp
+
+function echo_date() {
+  echo "$(date -u --rfc-3339=seconds) - $*"
+}
+
+echo_date "Checking/installing yq..."
+if ! [ -x "$(command -v yq4)" ]; then
+  wget -q -O /tmp/yq4 https://github.com/mikefarah/yq/releases/download/v4.34.1/yq_linux_amd64
+  chmod u+x /tmp/yq4
+fi
+which yq4
```

Comment:
Do we have a way to do this that doesn't hardcode the architecture? We have multi-arch `yq` container images that we use in the `upi-installer` image: https://github.com/openshift/installer/blob/master/images/installer/Dockerfile.upi.ci#L13-L14. Maybe they can be used here instead of `curl`ing the binary?

Reply:
@r4f4 that's a good suggestion. We should not use fixed arch here. I just changed to use the `upi-installer` from pre-built ones to consume the yq. Running reharsals.